app/export.go

```diff
@@ -249,10 +249,10 @@ func (a *App) ExportAllUsers(writer io.Writer) *model.AppError {
 	return nil
 }
 
-func (a *App) buildUserTeamAndChannelMemberships(userId string) (*[]UserTeamImportData, *model.AppError) {
+func (a *App) buildUserTeamAndChannelMemberships(userID string) (*[]UserTeamImportData, *model.AppError) {
```

Comment:
We are using `userId` everywhere, we prefer to stick with that until we change everything at the same time. Do you mind to revert that part of the change?

Reply:
Yes, sorry. It is such an automatism for me at this point. Makes sense to keep the code base consistent.